python/python/bystro/supervised_ppca/tests/test_utils_pt.py

```diff
@@ -1,4 +1,7 @@
+import random
 import torch
+import numpy as np
```

Comment:
The changes in this file (setting seeds) are necessary to make the test consistently pass in GitHub actions.